README.md

```diff
@@ -30,6 +30,7 @@ In trying to keep this template as generic and reusable as possible, there are s
 - [ ] Delete the content of [`CHANGELOG.md`](CHANGELOG.md). We encourage you to [keep a changelog](https://keepachangelog.com/en/1.0.0/).
 - [ ] Replace the generic content in this file with the relevant details about your project
 - [ ] Delete this section of the README
+- [ ] Acknowledge that some features like [branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule?source=post_page---------------------------) are only available when the repo is `public`
```

Comment:
Can you move this above the "Delete this section" checkbox so that's always the final step?

Can probably drop the `?source=` bit from the end of the URL btw

Reply:
moved up and delet source